components/org.wso2.carbon.appmgt.hostobjects/src/main/java/org/wso2/carbon/appmgt/hostobjects/APIProviderHostObject.java

```diff
@@ -375,19 +383,19 @@ public static NativeObject jsFunction_getBusinessOwnerCustomProperties(Context c
 
         NativeObject businessOwnerDataObject = new NativeObject();
         APIProvider apiProvider = getAPIProvider(thisObj);
-        Map<String, String> businessOwnerProperties = null;
-        businessOwnerProperties = apiProvider.getBusinessOwnerCustomProperties(businessOwnerId);
+        List<BusinessOwnerProperty> businessOwnerPropertiesList = apiProvider
+                .getBusinessOwnerCustomProperties(businessOwnerId);
         JSONObject businessOwnerPropertiesObject = new JSONObject();
-
-        Set<Map.Entry<String, String>> entries = businessOwnerProperties.entrySet();
-        for (Map.Entry entry : entries) {
-            String key = (String) entry.getKey();
-            String value = (String) entry.getValue();
-            businessOwnerPropertiesObject.put(key, value);
+        if(businessOwnerPropertiesList != null) {
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/4642888b8ad6934dcda162142607b8549b0728e5